src/Tests/dotnet-workload-list.Tests/GivenDotnetWorkloadList.cs

```diff
@@ -41,7 +41,7 @@ public void GivenNoWorkloadsAreInstalledListIsEmpty()
             command.Execute();
 
             // Expect 3 lines for table headers
```

Comment:
Update the comment to 5 or maybe change to something neutral like "Expected number of lines for table headers" in case we change this again

Reply:
Good catch, thanks!